### DISCUSSION TITLE
What has changed 217 -> 219

### DISCUSSION THREAD

**Discussion by Monberg75** [winder/Universal-G-Code-Sender#2620] [Help]
It would be really nice, if it was informed what was reset to default, so that we could know if it was relevant or not.

![image](https://github.com/user-attachments/assets/7d600db6-8b6e-4277-8137-7ca584225059)

**Comment by breiler** [comment]
I agree, this dialog is not very useful to a user from a UX perspective. We should try to rework this so we don't need to write over the old settings.

This dialog only effects these settings and is usually safe for the normal user:
![image](https://github.com/user-attachments/assets/ee8c689e-151c-4643-a202-d5f3362819d8)
